tests/zfs-tests/tests/functional/tmpfile/tmpfile_001_pos.c

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <string.h>
+
```

Comment:
It would be useful to add a comment that these values should be defined in `fcntl.h`.